tests/functional/DatadirTest.php

```diff
@@ -493,6 +493,43 @@ public function testCastOperatorQuery(): void
         $this->runAppWithConfig($config);
     }
 
+    public function testIntColumn(): void
+    {
+        // phpcs:disable Generic.Files.LineLength
+        $config = [
+            'authorization' => $this->getDatabaseConfig(),
+            'parameters' => [
+                'blocks' => [
+                    [
+                        'name' => 'first block',
+                        'codes' => [
+                            [
+                                'name' => 'first code',
+                                'script' => [
+                                    'DROP TABLE IF EXISTS "accounts";',
+                                    'CREATE TABLE "accounts" ("1234" int);',
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+            'storage' => [
+                'output' => [
+                    'tables' => [
+                        [
+                            'source' => 'accounts',
+                            'destination' => 'out.c-my.accounts',
+                        ],
+                    ],
+                ],
+            ],
+        ];
+        // phpcs:enable
+
+        $this->runAppWithConfig($config);
```

Comment:
Pridal by som assert na to, ako vyzera manifest ... aspon nazvy stlpcov. 
Ze to nepada je super, a este lepsie je vidiet, ze to uklada spravne,